### DISCUSSION TITLE
Add EnableCompressionInSingleFile to DotNetCorePublishSettings

### DISCUSSION THREAD

**Discussion by SIkebe** [cake-build/cake#3606] [Ideas]
`EnableCompressionInSingleFile` is not yet included in `DotNetCorePublishSettings`.

>## Compress assemblies in single file app
>Starting with .NET 6, single file apps can be created with compression enabled on the embedded assemblies. Set EnableCompressionInSingleFile property to true to achieve this. The produced single-file will have all of the embedded assemblies compressed which can significantly reduce the size of the executable. Compression comes with a performance cost. On application start, the assemblies must be decompressed into memory, which takes some time. It's recommended to measure both the size impact and startup cost impact of enabling compression before using it as the impact varies a lot between different applications.
https://docs.microsoft.com/en-us/dotnet/core/deploying/single-file#compress-assemblies-in-single-file-app

**Comment by SIkebe** [comment]
I've already implemented here.
https://github.com/SIkebe/cake/commit/38da45ebd82bd7ffc08cccc3926fb374715feac2

**Comment by augustoproiete** [comment]
Thanks @SIkebe, tracking via https://github.com/cake-build/cake/issues/3607 we'd take a PR, feel free to add a comment there if you'd like to send it yourself.
